### ISSUE TITLE
Import cover: La Campana South Facing 2016-2019

### ISSUE THREAD

**Issue by KW529** [felt0134/drought-net-tickets#338]

**Comment by KW529** [comment]
Hi Katja and Liesbeth,

Happy New Year! I have a few questions about the cover data from the La Campana and Quebrada de Talca sites. First, could you all please add information to the taxa table for the attached taxa ('MissingTaxa_LC-QDT.xlsx')? I have also attached the taxa table you all originally submitted for these sites. 

In addition, I found the following duplicate values (also see tables below): 
Dioscorea sp.1 in plot 13 for 2017 at La Campana South 

site | plot | subplot | taxa | cover | date
-- | -- | -- | -- | -- | --
La Campana South Facing | 13 | total | Dioscorea sp1. | <1 | 22/8/2017
La Campana South Facing | 13 | total | Dioscorea sp1. | <1 | 22/8/2017

Should I erase one of the duplicates or add these together?

Thanks so much for any clarification!
[MissingTaxa_LC-QDT.xlsx](https://github.com/KW529/drought-net-tickets/files/7831348/MissingTaxa_LC-QDT.xlsx)

**Comment by KW529** [comment]
Dear Kate,

Happy New Year to you too! It is good to hear from you.

I have gone through your files carefully, and added the missing taxa to the taxa file (la_campana_quebrads_taxa_revised_LvdB).
There were a few taxa in the MissingTaxa_LC_QDT file that were in the taxa list, but had been under a corrected name. I added those names in an extra column in the MissingTaxa_LC_QDT_revised_LvdB    file as ="....".
The species that were only identified until genus (sp.) are all unique, meaning that they are the same on the north and south facing slope in La Campana (La Campana North and La Campana South) if they do not differ in name.
I added this info to the MissingTaxa_LC_QDT_revised_LvdB file as The same as on La Campana North.
I found that Foeniculum vulgare was 3x in the la_campana_quebrads_taxa and deleted 2 of them.
I added a few local names
I saw some errors in the collectors names, that I corrected (mainly letters with accent, that I replaced for letters without accent)

About the duplicates: great catch. These should only be once in the table, so one of the duplicates should be erased. However, there should be Stachys macraei in plot 3 and plot 5 (one entry in each).

Let me know if there are any uncertainties!

Cheers,
Liesbeth
